src/Catty/Helpers/Util/Util.swift

```diff
@@ -88,11 +118,57 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         let platformVersionWithPatch = "\(major).\(minor).\(patch)" as String
         return platformVersionWithPatch
     }
+
     class func platformVersionWithoutPatch() -> String {
         let os = self.platformVersion()
         let major = String(format: "%ld", os.majorVersion)
         let minor = String(format: "%ld", os.minorVersion)
         let platformVersionWithoutPatch = "\(major).\(minor)" as String
         return platformVersionWithoutPatch
     }
+
+    class func showNotification(withMessage message: String?) {
+        guard let hud = BDKNotifyHUD(image: nil, text: message) else {
+            return
+        }
+
+        let vc = Util.topmostViewController()
+        if vc.view == nil {
+            return
+        }
+
+        hud.destinationOpacity = CGFloat(kBDKNotifyHUDDestinationOpacity)
+        hud.center = CGPoint(x: vc.view.center.x, y: vc.view.center.y)
+
```

Comment:
You forgot to add `vc.view.addSubview(hud)` here.

Reply:
Thanks! I have now added the missing line.